src/labels/CustomLabels.labels

```diff
@@ -3246,6 +3246,22 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>alloUnallocated</shortDescription>
         <value>unallocated</value>
     </labels>
+    <labels>
+        <fullName>bdeRetirementBlockerHeader</fullName>
+        <categories>Batch Data Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Header about Batch Data Entry being unsupported</shortDescription>
+        <value>Batch Data Entry Not Supported</value>
+    </labels>
+    <labels>
+        <fullName>bdeRetirementBlockerMessage</fullName>
+        <categories>Batch Data Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Explanation of Batch Data Entry not being supported &amp; recommendations</shortDescription>
+        <value>As of October 1, 2020, Batch Data Entry is no longer supported. Most capabilities have been replaced with newer features in NPSP. If you have any in-progress batches, you can still access those records, but you can&apos;t process them using Batch Data Entry. &lt;br/&gt; Contact support if you need to access in-progress batches. You may want to explore Gift Entry as an alternative to Batch Data Entry. Learn more about Gift Entry in the &lt;a href=&apos;https://powerofus.force.com/NPSP_Documentation&apos; target=&apos;_blank&apos;&gt;NPSP Documentation&lt;/a&gt; on the Power of Us Hub.</value>
```

Comment:
```suggestion
        <value>As of October 1, 2020, Batch Data Entry is no longer supported. Most capabilities have been replaced with newer features in NPSP. If you have any in-progress batches, you can still access those records, but you can't process them using Batch Data Entry. Contact support if you need to access in-progress batches. &lt;br/&gt; You may want to explore Gift Entry as an alternative to Batch Data Entry. Learn more about Gift Entry in the &lt;a href=&apos;https://powerofus.force.com/NPSP_Documentation&apos; target=&apos;_blank&apos;&gt;NPSP Documentation&lt;/a&gt; on the Power of Us Hub.</value>
```

Reply:
@voduyemi I added one more suggested edit here to clean up a rich text apostrophe and to move the line break. Let me know if you have any questions. Thanks!

Reply:
Updated!